chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestDefinition.java

```diff
@@ -100,6 +104,17 @@ public Builder withQueryProvider(Iterable queryProvider) {
         }
 
         /**
+         * Optional: Sets load balancer (subtypes of {@link QueryPoolLoadBalancer}).
+         * Default is {@link RoundRobinLoadBalancer}
+         *
```

Comment:
Is TestDefinitionGenerator a client class?
I think, that JTestDefinition is just a configuration and it must not set any defaults by itself. And TestDefinitionGenerator seems to be perfect place to set it. 
Can you explain why it should be in JTestDefinition?